utils/__init__.py

```diff
@@ -710,9 +730,12 @@ def biosimulators_core(engine,omex_filepath,output_dir=None):
     output_dir: folder to write the simulation outputs to
     '''
 
+    omex_filepath_no_spaces = remove_spaces_from_filename(omex_filepath)
```

Comment:
Are you certain it's spaces in the omex filename that cause problems, what about the sedml and sbml filenames?

Reply:
Yes, it is the mounting of the omex file into Docker that is causing the issues (the SED-ML and SBML files are not mounted)